ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -811,11 +873,18 @@ def _on_finish_session(finish_args: CISession.FinishArgs):
     session.finish(finish_args.force_finish_children, finish_args.override_status)
 
 
+@_requires_civisibility_enabled
+def _on_get_session_settings(session_id: CISessionId):
+    log.debug("Handling get session settings for session id %s", session_id)
+    return CIVisibility.get_session_settings(session_id)
+
+
 def _register_session_handlers():
     log.debug("Registering session handlers")
-    core.on("ci_visibility.session.register", _on_discover_session)
+    core.on("ci_visibility.session.discover", _on_discover_session)
```

Comment:
Just for my context, why did this change?

Reply:
Just to keep in line with the others. I originally had everything as `register` , then switched to `discover` because that felt more correct to me.